### PR TITLE
Analytics 9.0.0

### PR DESCRIPTION
**Pull request by tsunghung**
Analytics 9.0.0 release for CocoaPods and Swift Package Manager.

### CODE REVIEW

File: FirebaseAnalytics.podspec.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "z"
     ],
     "license": {
-        "text": "Copyright 2021 Google",
+        "text": "Copyright 2022 Google",
```
Comment (paulb777) [inline FirebaseAnalytics.podspec.json:25]: Technically Copyrights only need to be updated when the file is created, but we have been updating them for podspecs - so either way.

File: SwiftPMTests/analytics-import-test/analytics-import.swift [hunk 2]
```diff
@@ -21,7 +21,7 @@ import FirebaseAnalytics
```
Comment (paulb777) [inline SwiftPMTests/analytics-import-test/analytics-import.swift:21]: Consider adding an import to the new library in this test and any other possible tests.

**Review by paulb777 — APPROVED** [review]

File: FirebaseAnalytics.podspec.json (hunk 1 above)
Comment (tsunghung) [reply FirebaseAnalytics.podspec.json:25]: Interesting, thanks for the info.

File: SwiftPMTests/analytics-import-test/analytics-import.swift (hunk 2 above)
Comment (tsunghung) [reply SwiftPMTests/analytics-import-test/analytics-import.swift:21]: The new library test is tricky. The API is exported in FirebaseAnalytics. Without the OnDeviceConversion library, the new API will still be visible, but just an no-op. To verify the internal behavior is also difficult here. Will discuss internally to see how to create tests here.

**Pull request merged**